test_e2e/check-diff-expected-output.py

```diff
@@ -28,6 +28,7 @@ def check_files(expected_output_dir: Path, current_output_dir: Path):
     diff = set(expected_files) ^ set(current_files)
     if diff:
         print("diff in files that exist: {}".format(", ".join(diff)))
+        print(f"expected files {expected_files}\nFiles obtained {current_files}")
```

Comment:
Was this here for debugging? Did the original print statement not enough? Just making sure.

Reply:
Exactly, that's for debugging. I found the orignal info less readable when debugging, but that's just personal preference. I love to physically see both sets to compare. When I initially had a problem that one of the sets was empty I was getting a list of files that I wasn't sure which set it actualy came from and which was empty.  Turns out it was example data set that was empty not my output, but it wasn't clear for me until I printed the sets. So I figured no harm in leaving it here, maybe it wll help someone else too.